lib/tasks/data_migration.rake

```diff
@@ -76,4 +76,15 @@ namespace :data_migration do
       puts "Updated #{args[:key]} in #{list.title} to #{new_criteria}"
     end
   end
+
+  desc "Fetch descriptions for single page subscription lists which don't have descriptions"
+  task fetch_subscriber_list_descriptions: :environment do |_t, _args|
+    SubscriberList.where.not(content_id: nil).where(description: nil).find_each do |list|
+      new_description = Services.content_store.content_item(list.url).to_h["description"]
```

Comment:
`.to_h` isn't needed here, the [`GdsApi::Response` delegates `:[]`](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/response.rb#L95).